src/main/java/mate/academy/spring/mapper/impl/request/ShoppingCartRequestMapper.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.mapper.impl.request;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.ShoppingCartRequestDto;
+import mate.academy.spring.service.TicketService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartRequestMapper implements
+        DtoRequestMapper<ShoppingCartRequestDto, ShoppingCart> {
```

Comment:
We do not use it
![image](https://user-images.githubusercontent.com/20326858/234532110-6beb5260-ed5d-4075-9082-f792b535ce7a.png)